.ci/pitest.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Attention, there is no "-x" to avoid problems on CircleCI
+set -e
+
+case $1 in
+
+pitest-annotation \
+|pitest-ant \
+|pitest-api \
+|pitest-blocks \
+|pitest-coding-1 \
+|pitest-coding-2 \
+|pitest-coding-require-this-check \
+|pitest-common \
+|pitest-common-2 \
+|pitest-design \
+|pitest-filters \
+|pitest-gui \
+|pitest-header \
+|pitest-imports \
+|pitest-indentation \
+|pitest-java-ast-visitor \
+|pitest-javadoc \
+|pitest-main \
+|pitest-metrics \
+|pitest-misc \
+|pitest-modifier \
+|pitest-naming \
+|pitest-packagenamesloader \
+|pitest-regexp \
+|pitest-sizes \
+|pitest-tree-walker \
+|pitest-utils \
+|pitest-whitespace \
+|pitest-xpath)
+  mvn -e --no-transfer-progress -P"$1" clean test-compile org.pitest:pitest-maven:mutationCoverage
+  groovy ./.ci/pitest-survival-check-xml.groovy "$1"
+  ;;
+
+# pitesttyle-gui)
+#   mvn -e -P$1 clean test-compile org.pitest:pitest-maven:mutationCoverage;
+#   # post validation is skipped, we do not test gui throughly
+#   ;;
+
+--list)
+  echo "Supported profiles:"
+  pomXmlPath="$(dirname "${0}")/../pom.xml"
+  sed -n -e 's/^.*<id>\(pitest-[^<]\+\)<\/id>.*$/\1/p' < "${pomXmlPath}" | sort
+  ;;
+
+*)
```

Comment:
Why have separate logic for this? The groovy script has `--list` functionality.

Reply:
it has bad output:
```
$ groovy ./.ci/pitest-survival-check-xml.groovy --list
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.reflection.CachedClass$3$1 (file:/opt/groovy/groovy-2.4.7/lib/groovy-2.4.7.jar) to method java.lang.Object.finalize()
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.reflection.CachedClass$3$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Supported profiles:
....
```

better to not bother user with all this noise.

Reply:
Resolved